client/app/lib/providers/computecontroller.coffee

```diff
@@ -997,6 +999,47 @@ module.exports = class ComputeController extends KDController
           @emit 'StacksInconsistent', stack
 
 
+  sharingStackTemplate: (_id, type) ->
+    remote.api.JStackTemplate.one { _id }, (err, template) ->
+      template.setAccess type
```

Comment:
yes, `sharingStackTemplate` tells something ongoing but we need to put the action into method name not the state.